roles/pcp/README.md

```diff
@@ -101,10 +101,10 @@ Central PCP setup for monitoring of several remote hosts.
   roles:
     - role: performancecopilot.metrics.pcp
       vars:
-        pcp_pmlogger_interval: 10
-        pcp_pmlogger_discard: 5
-        pcp_target_hosts: [slip, slop, slap]
-        pcp_rest_api: true
+    ### pcp_pmlogger_interval: 10
+    ### pcp_pmlogger_discard: 5
+    ### pcp_target_hosts: [slip, slop, slap]
+    ### pcp_rest_api: true
```

Comment:
not sure what markdownlint was complaining about here, but these should not be commented out

Reply:
Agreed, this looks like the same issue as above and the change seems incorrect here.